admin/lib/workarea/api/admin/engine.rb

```diff
@@ -14,10 +14,12 @@ class Engine < ::Rails::Engine
 
         config.after_initialize do
           Workarea::Api::Admin::Swagger.generate!
+          Workarea::Api::Admin::DateIndexes.load
```

Comment:
This won't be necessary since it's in `to_prepare`, which gets run on boot if eager loading and on each request otherwise.

Reply:
it didn't work without this